backend/config.js

```diff
@@ -1,34 +1,34 @@
 // Import the functions you need from the SDKs you need
-import { initializeApp } from 'firebase/app';
-import { getDatabase } from 'firebase/database';
+import { initializeApp } from "firebase/app";
```

Comment:
None of the changes here should be new. Run `git merge main` from your branch to get all the new changes. This will help with merge conflicts as well. Please let me know if there are conflicts that you are unsure of, rather than just guessing